src/core/server/rendering/rendering_service.tsx

```diff
@@ -164,4 +166,12 @@ export class RenderingService {
         return false;
       });
   };
+
+  public checkTitleValid = (title: string, configName?: string): boolean => {
+    if (!title || title.length > 36) {
```

Comment:
this will be important to call out. I would make sure to add this  to the documentation and log a specific warning about the character length.

Reply:
one thing that is good is that im not able to render HTML in this but im still able to render emojis. which i dont think is a security concern nor am i positive we can do anything about it if we want to allow multiple types of characters.

Reply:
Thanks for testing the title config with different scenarios~ Yea I think maybe we can specifically call out in the document that user should only input a string of letters that represents application title for the title config. 

Also for title length, i have updated the warning to include saying that title length should be between 1 to 36 characters.